gremlin-go/driver/request.go

```diff
@@ -83,6 +88,85 @@ func makeBytecodeRequest(bytecodeGremlin *Bytecode, traversalSource string, sess
 	}
 }
 
+// allowedReqArgs contains the arguments that will be extracted from the
+// bytecode and sent with the request.
+var allowedReqArgs = map[string]bool{
+	"evaluationTimeout":       true,
+	"scriptEvaluationTimeout": true,
```

Comment:
i wouldn't bother supporting `scriptEvaluationTimeout` - it's a deprecated argument where `evaluationTimeout` is preferred.

Reply:
Thanks for the review!

Would it be a problem to keep it for now? If I'm not wrong, this option is still supported by other GLVs and mentioned in the Tinkerpop reference. By supporting it we anticipate to users that try to use the old name and don't understand why the timeout is being ignored or why it does not works with all GLVs. I feel it is better to be consistent among GLVs and, once the setting is fully deprecated, removing the support from all the languages at once. What do you think?

Some instances,

**gremlin-python**

https://github.com/apache/tinkerpop/blob/7ad487b75992a4c46871ae9aab2d5451c133eb8b/gremlin-python/src/main/python/gremlin_python/driver/driver_remote_connection.py#L171

**Tinkerpop reference**

In the different GLV sections:

> The following options are allowed on a per-request basis in this fashion: batchSize, requestId, userAgent and evaluationTimeout (formerly scriptEvaluationTimeout which is also supported but now deprecated).

Which suggests that `scriptEvaluationTimeout` should still work although is not recommended to use it.

Reply:
`scriptEvaluationTimeout` was technically removed in 3.5.0: https://issues.apache.org/jira/browse/TINKERPOP-2295 if there are remnants of it still around, that is probably a mistake. there are some graphs that quietly still support it in 3.5.x, but i don't think TinkerPop needs to continue to quietly do the same. and gremlin-go is new so why bother to allow folks to write code that uses deprecated/removed things. Just seems like we'd be adding things to forget to remove later 🙂

Reply:
That makes sense. I removed `scriptEvaluationTimeout` both from the gremlin-go docs and code. I didn't touch any other GLV because it feels like it should go in a different PR.

eb9b358 should fix it. PTAL :slightly_smiling_face: 

Thanks!